collect_app/src/main/java/org/odk/collect/android/formmanagement/BlankFormsListViewModel.java

```diff
@@ -87,6 +99,12 @@ public void syncWithServer() {
         });
     }
 
+    private void logManualSync() {
+        String url = preferencesProvider.getGeneralSharedPreferences().getString(GeneralKeys.KEY_SERVER_URL, "");
+        String urlHash = FileUtils.getMd5Hash(new ByteArrayInputStream(url.getBytes()));
```

Comment:
What do you think of using the hostname instead of the raw setting value? The raw server setting value would mean we differentiate between different Central App Users. Hostname would give us a sense of organizations that use match exactly and specifically the manual sync button.

Reply:
Yeah that makes more sense